CHANGELOG.md

```diff
@@ -5,31 +5,33 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 
-## Unreleased as of Sprint 96 ending 2018-10-08
+## Unreleased as of Sprint 97 ending 2018-10-22
 
 ### Added
-- Compress Azure HyperV and EC2 images [(#289)](https://github.com/ManageIQ/manageiq-appliance-build/pull/289)
-- Add post compression support [(#288)](https://github.com/ManageIQ/manageiq-appliance-build/pull/288)
-- Move from apache module mod_auth_kerb to mod_auth_gssapi [(#282)](https://github.com/ManageIQ/manageiq-appliance-build/pull/282)
-
-### Fixed
-- Set commit_sha just once [(#290)](https://github.com/ManageIQ/manageiq-appliance-build/pull/290)
+- Add pylxca python module for Lenovo [(#293)](https://github.com/ManageIQ/manageiq-appliance-build/pull/293)
+- Add vspk python module for Nuage [(#292)](https://github.com/ManageIQ/manageiq-appliance-build/pull/292)
 
-## Unreleased as of Sprint 95 ending 2018-09-24
+## Hammer Beta-1 - Released 2018-10-12
 
 ### Added
 - Add RPMs needed for Ansible playbooks and roles for oVirt [(#281)](https://github.com/ManageIQ/manageiq-appliance-build/pull/281)
 - Update appliances to ruby 2.4.4 [(#280)](https://github.com/ManageIQ/manageiq-appliance-build/pull/280)
-
-## Unreleased as of Sprint 94 ending 2018-09-10
+- Add support for OpenID-Connect [(#272)](https://github.com/ManageIQ/manageiq-appliance-build/pull/272)
+- Add sqlite-devel package [(#260)](https://github.com/ManageIQ/manageiq-appliance-build/pull/260)
+- Remove enable for docker service [(#251)](https://github.com/ManageIQ/manageiq-appliance-build/pull/251)
+- Add the docker package to run awx locally on the appliance [(#250)](https://github.com/ManageIQ/manageiq-appliance-build/pull/250)
+- Add post compression support [(#288)](https://github.com/ManageIQ/manageiq-appliance-build/pull/288)
+- Compress Azure, HyperV and EC2 images [(#289)](https://github.com/ManageIQ/manageiq-appliance-build/pull/289)
+- Move from apache module mod_auth_kerb to mod_auth_gssapi [(#282)](https://github.com/ManageIQ/manageiq-appliance-build/pull/282)
 
 ### Fixed
 - Update vSphere ova [(#278)](https://github.com/ManageIQ/manageiq-appliance-build/pull/278)
+- Increase filesystem sizes [(#232)](https://github.com/ManageIQ/manageiq-appliance-build/pull/232)
+- Set commit_sha just once [(#290)](https://github.com/ManageIQ/manageiq-appliance-build/pull/290)
 
-## Unreleased as of Sprint 91 ending 2018-07-30
+### Removed
+- Remove non-existing "server-policy" yum group [(#268)](https://github.com/ManageIQ/manageiq-appliance-build/pull/268)
```

Comment:
This line should be removed as this fix first appeared in Gaprindashvili-4.

Reply:
How did it even get here? This wasn't in an "unreleased" section before this PR ...